doc: update documentation and definitions files (#154)
dashevo/wallet-lib: docs/account/createTransaction.md

Notes: This transaction will be need to be signed [`account.sign(transaction)`](/account/sign) and then, if wanted, broadcasted to the network for execution `account.broadcastTransaction()`.

Example : 
```js
const recipients = [{recipient:"yereyozxENB9jbhqpbg1coE5c39ExqLSaG", satoshis:10e8},{recipient: "yMN2w8NiwcmY3zvJLeeBxpaExFV1aN23pg", satoshis: 1e8}];
const change = "yaVrJ5dgELFkYwv6AydDyGPAJQ5kTJXyAN";
const tx = account.createTransaction({recipients, change});
```

**Strategy:**

By default, wallet-lib is shipped with two different strategies : 

> - **simpleDescendingStrategy** : Will try to use as much small inputs as possible up to meeting the necessary amount required.  

Reviewer: ```suggestion
- **simpleDescendingStrategy** : Will try to use as many small inputs as possible to meet the amount required.  
```

>     Allow fee to be optimized for the smallest size at the cost of breaking big inputs in smaller size.

Reviewer: ```suggestion
    Allows the fee to be optimized for the smallest size at the cost of breaking big inputs.
```

> - **simpleAscendingStrategy** : Will try to use as much small inputs as possible up to meeting the necessary amount required.  

Reviewer: ```suggestion
- **simpleAscendingStrategy** : Will try to use as many small inputs as possible to meet the amount required.  
```

>     Allow to get rid of many small inputs at the cost of potential bigger fee.

Reviewer: ```suggestion
    Allows using many small inputs at the cost of a potentially bigger fee.
```

You can also pass your own strategy (as a function) to allow you to create your own strategy for how you will want to spend the UTXO.   

```js
const recipient = "yereyozxENB9jbhqpbg1coE5c39ExqLSaG";
const satoshis = 10e8;
const specialStrategy = (utxosList, outputsList, deductFee = false, feeCategory = 'normal')=> { 
//...
};
const txOpts1 = {
     recipient, satoshis,

>      strategy: 'simpleAscensingStrategy',

Reviewer: ```suggestion
     strategy: 'simpleAscendingStrategy',
```

Author: Wow ! Thank you so much ! Eagle eyes on this *.* <3

By default, wallet-lib is shipped with two different strategies : 

> - **simpleDescendingStrategy** : Will maximize the uses of big inputs to meet the amount required.  

Reviewer: ```suggestion
- **simpleDescendingStrategy** : Will maximize the use of big inputs to meet the amount required.  
```
